examples/async_stream/main.cc

```diff
@@ -15,7 +16,16 @@ int main()
                     std::thread([stream =
                                      std::shared_ptr<drogon::ResponseStream>{
                                          std::move(stream)}]() mutable {
-                        std::cout << std::boolalpha << stream->send("hello ")
+                        static std::string str{"hello"};
+                        auto dest = new char[20]();
+                        memcpy(dest, str.c_str(), str.size());
+                        std::shared_ptr<char> auto_free(dest, [](char *p) {
+                            delete[] p;
+                        });
```

Comment:
There's an overload for `shared_ptr` and `unique_ptr` which does exactly this...

```
std::shared_ptr<char[]> dest { new char[20] };
```
And `memcpy` in c++ code is... bold (even `strcpy` wouldn't be better).

Reply:
Actually, this is just an example